### ISSUE TITLE
Failed to resolve: com.github.MKergall:osmbonuspack:6.5.2

### ISSUE THREAD

**Issue by outdoorapps** [MKergall/osmbonuspack#351]
Used this on gradle:
implementation 'com.github.MKergall:osmbonuspack:6.5.2'

Got:
Failed to resolve: com.github.MKergall:osmbonuspack:6.5.2

Any idea what is happening? Thanks.

**Comment by MKergall** [comment]
Follow https://github.com/MKergall/osmbonuspack/wiki/HowToInclude#1-including-using-jitpack


dependencies {
    ...
    compile 'com.github.MKergall:osmbonuspack:6.5.2'
}

**Comment by MKergall** [comment]
ping...

**Comment by MKergall** [comment]
6.5.3 now available

**Comment by selphiron** [comment]
I have the same problem. I tried these things:
`compile 'com.github.MKergall:osmbonuspack:6.5.2'`
Error:

> Configuration 'compile' is obsolete and has been replaced with 'implementation' and 'api'.
It will be removed at the end of 2018. For more information see: http://d.android.com/r/tools/update-dependency-configurations.html
Failed to resolve: com.github.MKergall:osmbonuspack:6.5.2
Show in File
Show in Project Structure dialog

When I try
`api 'com.github.MKergall:osmbonuspack:6.5.2'` or
`implementation 'com.github.MKergall:osmbonuspack:6.5.2'`
Error:

> Failed to resolve: com.github.MKergall:osmbonuspack:6.5.2
Show in File
Show in Project Structure dialog

I tried the same thing with 6.5.3 and got the same errors.
